src/test/countComments.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import countComments from '../modules/countComments';
+
+describe('Count comments', () => {
+  beforeAll(() => {
+    document.body.innerHTML = ` <ul class="ul-comment">
+        <li>1</li>
+        <li>3</li>
+        <li>5</li>
+       </ul>`;
+  });
+
+  test('it should print 3', () => {
+    const ul = document.querySelector('.ul-comment');
+    const result = countComments(ul);
+    expect(result).toBe(3);
+  });
+});
```

Comment:
- [ ] Good job implementing a test for the counter function. Please make sure that you have tests for all edge cases like when there are 0 items on the page (counter function should return 0).

Reply:
@ShadyShawkat, 
I have completed the implementation of the test and have pushed the update. Please review and approve. Thank you.